crates/core/tedge_api/src/alarm.rs

```diff
@@ -128,20 +129,40 @@ mod tests {
         json!({
             "severity": "high",
             "text": "I raised it",
-            "time": "2021-04-23T19:00:00+05:00",
+            "time": 1701954000,
         }),
         ThinEdgeAlarm {
             alarm_type: "temperature_alarm".into(),
             source: EntityTopicId::default_main_device(),
             data: Some(ThinEdgeAlarmData {
                 severity: Some("high".into()),
                 text: Some("I raised it".into()),
-                time: Some(datetime!(2021-04-23 19:00:00 +05:00)),
+                time: Some(datetime ! (2023 - 12 - 07 13: 00: 00 + 00: 00)),
                 extras: hashmap!{},
             }),
         };
         "alarm parsing for main device"
     )]
+    #[test_case(
+    "temperature_alarm",
+    EntityTopicId::default_main_device(),
+    json ! ({
+    "severity": "high",
+    "text": "I raised it",
+    "time": 1701954000,
+    }),
+    ThinEdgeAlarm {
+    alarm_type: "temperature_alarm".into(),
+    source: EntityTopicId::default_main_device(),
+    data: Some(ThinEdgeAlarmData {
+    severity: Some("high".into()),
+    text: Some("I raised it".into()),
+    time: Some(datetime ! (2023 - 12 - 07 13: 00: 00 + 00: 00)),
+    extras: hashmap ! {},
+    }),
+    };
+    "alarm parsing with unix timestamp"
+    )]
```

Comment:
I don't quite know what I messed up here with the indentation, but I'm just leaving a note here so I remember to fix it